backend/src/routes/v2/schema/getSchemas.ts

```diff
@@ -19,52 +21,13 @@ interface GetSchemaResponse {
 export const getSchemas = [
   bodyParser.json(),
   async (req: Request, res: Response<GetSchemaResponse>) => {
-    return res.json({
-      data: {
-        schemas: [
-          {
-            id: 'example-schema-1',
-            name: 'Example Schema 1',
-            description: 'This is a description of the schema.',
-
-            active: true,
-            hidden: false,
-
-            kind: 'deployment',
-            meta: { example: true },
-
-            uiSchema: {
-              'UI Schema field 1': 'field 1 info',
-            },
-            schema: {
-              'Schema field 1': 'field 1 info',
-            },
+    const { query } = parse(req, getSchemasSchema)
 
-            createdAt: new Date(),
-            updatedAt: new Date(),
-          },
-          {
-            id: 'example-schema-2',
-            name: 'Example Schema 2',
-            description: '',
+    const schemas: Array<SchemaInterface> = await findSchemasByKind(query.kind)
```

Comment:
Unnecessary typing.  If TypeScript can't work it out for itself, you should add a return type to `findSchemasByKind` instead of putting it here.